sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestManager.java

```diff
@@ -506,7 +506,7 @@ public void write(final ChannelHandlerContext context, final Object message, fin
 
         if (message == RntbdHealthCheckRequest.MESSAGE) {
 
-            context.write(RntbdHealthCheckRequest.MESSAGE, promise).addListener(completed -> {
+            context.writeAndFlush(RntbdHealthCheckRequest.MESSAGE, promise).addListener(completed -> {
```

Comment:
why `flush` is required here? I don't think this is in the scope of the connection listener. am I right?

please undo this if unrelated to this PR.

Reply:
Has removed this change to PR https://github.com/Azure/azure-sdk-for-java/pull/16197